src/twisted/web/test/test_web.py

```diff
@@ -957,6 +957,28 @@ def getContentFile(length):
         self.assertEqual([12345], lengths)
         self.assertIs(contentFile, request.content)
 
+    def test_parseBodySetFromSite(self):
```

Comment:
New tests should be annotated, unless it causes a truly obnoxious amount of additional annotation work:
```suggestion
    def test_parseBodySetFromSite(self) -> None:
```